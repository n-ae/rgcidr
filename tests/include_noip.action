-i 192.168.0.0/16 tests/mixed_with_noip.given
