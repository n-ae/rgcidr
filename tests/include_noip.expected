This line has no IP
Another line without IP
10.0.0.1
Pure text line
8.8.8.8
Final line no IP
